fair/ancil/units.csv

```diff
@@ -0,0 +1,163 @@
+"Default Units for Different Variables,	"

```

Comment:
can we make this csv separated rather than tab separated please?

Reply:
Done